### ISSUE TITLE
Is Transmission Officially 3.00 Supported by 5.18.0?

### ISSUE THREAD

**Issue by tool-maker** [transmission-remote-gui/transgui#1334]
1. Windows 10
2. Transmission Daemon from Transmission 3.00
3. Transmission Remote GUI 5.18.0

I encountered numerous issues when I tried the daemon from Transmission 3.00 with Transmission Remote GUI 5.18.0 on Windows 10.

Before reporting these or attempting to understand what the issues are, I would like to know whether we should even be trying to make this work with Transmission 3.00.

Thanks.

**Comment by Tharn** [comment]
Yes, it should work the same as it works with Transmission 2.9x. As in... there are some bugs. But I have not found any (I think) that are particular to using Transmission 3.0 with it.
